app-text/xlreader/xlreader-0.9.0.recipe

```diff
@@ -0,0 +1,40 @@
+SUMMARY="Excel document reader"
+DESCRIPTION="Read data from Excel 97+ spread sheets without Microsoft! \
+Provides an API to write applications that read Excel documents. The xlreader \
+tool converts data to tab delimited, CSV or SQL inserts. Written in C. Based on \
+the the Java version by Andrew Khan."
+HOMEPAGE="http://giffin.org/xlreader"
+COPYRIGHT="2001 David Giffin"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://downloads.sourceforge.net/xlreader/xlreader-$portVersion.tgz"
+CHECKSUM_SHA256="53d556a77d67b3d6e8a4919a7753c019fc5395b8e2f5e33eb324dcf7c075d7fa"
+SOURCE_DIR="xlreader-$portVersion"
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	xlreader = $portVersion
+	cmd:xlreader = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc
+	"
+
+BUILD()
+{
+	make
+}
+
+INSTALL()
+{
+	install -d $binDir
+	install -t $binDir xlreader
```

Comment:
We need to include the COPYRIGHT file.
There is also a README but it includes information for building and installing xlreader.
Since an easy sed command can remove that information, it might be nice to also include the resulting file.
What do you think?
```diff
-	install -d $binDir
+	install -d $binDir $docDir
 	install -t $binDir xlreader
+	install -t $docDir COPYRIGHT README
+	sed -i -e '/^INSTALL NOTES$/,/^HELP$/ d;' $docDir/README
```

Reply:
Hmm. "sed" seems pretty useful. I never knew about that.